Tasks/VsTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -40,6 +40,7 @@
   "loc.input.help.configuration": "Configuration against which the tests should be reported. If you have defined a variable for configuration in your build task, use that here.",
   "loc.input.label.publishRunAttachments": "Upload Test Attachments",
   "loc.input.help.publishRunAttachments": "Opt in/out of publishing test run level attachments.",
+  "loc.messages.PathDoesNotExist": "%s path does not exist.",
```

Comment:
>loc.messages.PathDoesNotExist": "%s path does not exist.", [](start = 3, length = 58)

check what happen in case of access denied. If this message gets printed then, we should modify message to "%s path either does not exist or access is denied to it "